### ISSUE TITLE
How could I find the training dataset?

### ISSUE THREAD

**Issue by zhren711** [ouyangbaicai/FusionGCN#2]

**Comment by ouyangbaicai** [comment]
There is no large publicly available standard training dataset in the field of MFIF. In this paper, we utilized the Pascal Visual Object Classes 2012 (VOC2012) to create a training dataset comprising 20 classes of common objects, such as people, dogs, cars, airplanes, and more. The process of creating the training set can be divided into five main steps: Gaussian filtering, image transformation, image inversion, elementwise multiplication, and element-wise addition. Initially, we take the source image and apply Gaussian filtering with various filter sizes to produce images with different degrees of blur. These blurred images are then used to mimic different bokeh effects. During the image transformation and inversion stages, we perform a binarization operation on the mask to convert it into training labels. We then globally invert these training labels to create complementary labels, which help in emphasizing the blurring of the foreground and background in the images. Finally, we carry out element-wise multiplication and addition with the training labels and the blurred images generated earlier to create multiple pairs of images. 

or

Send an email to my mailbox to add my contact information

**Comment by zhren711** [comment]
> 在 MFIF 领域没有大型的公开可用的标准训练数据集。在本文中，我们利用 Pascal Visual Object Classes 2012 （VOC2012） 创建了一个包含 20 类常见对象（如人、狗、汽车、飞机等）的训练数据集。创建训练集的过程可以分为五个主要步骤：高斯滤波、图像转换、图像反转、元素乘法和元素加法。最初，我们获取源图像并应用具有各种滤镜大小的高斯滤波，以生成具有不同模糊程度的图像。然后使用这些模糊图像来模拟不同的散景效果。在图像转换和反转阶段，我们对掩码执行二值化作，将其转换为训练标签。然后，我们将这些训练标签全局反转以创建互补标签，这有助于强调图像中前景和背景的模糊。最后，我们使用训练标签和之前生成的模糊图像进行元素乘法和加法，以创建多对图像。
> 
> 或
> 
> 向我的邮箱发送电子邮件以添加我的联系信息

Thanks for your comprehensive reply, it is a great work.

**Comment by zhren711** [comment]
Sorry, I have an extra question about the GCN-only result shown in Fig.1 (a). I wonder could you release the GCN-only architecture code in this experiment？

**Comment by ouyangbaicai** [comment]
We do not disclose all ablation network structures except for the baseline. For details, please refer to section 4.4.1 of the article. If you want to reproduce the result of Figure 1 (a), you only need to connect the network output layer to the GCN decoder, cut off the connection between GCN and CNN, and comment out the CNN part of the code.
